prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -41,47 +56,49 @@ def build_agent_functions(
     if allow_stop:
         functions.append(Stop())
 
+    functions.extend([f() for f in LEARNING_FUNCTIONS])
+    functions.extend([f(agent=agent) for f in AGENT_FUNCTIONS])
     functions.extend([f(market_type=market_type) for f in MARKET_FUNCTIONS])
     if market_type == MarketType.OMEN:
         functions.extend([f() for f in OMEN_FUNCTIONS])
     if long_term_memory:
         functions.append(
-            RememberPastLearnings(long_term_memory=long_term_memory, model=model)
+            RememberPastActions(long_term_memory=long_term_memory, model=model)
```

Comment:
It's the same thing, but renamed for clarity, as mentioned in the other comment.